uber/templates/preregistration/form.html

```diff
@@ -27,14 +27,14 @@
                 }
             });
         {% endif %}
```

Comment:
I think this is incorrect.  ``c.DONATION_TIERS`` is a dictionary mapping integers to descriptions, whereas ``c.PREREG_DONATION_OPTS`` is (I believe) a list of tuples.  I'd need to double check to be sure though.